### PR TITLE
cleanup formatting of image docs

### PR DESCRIPTION
**Pull request by tpoitras**
Trello cards:
- Perl: https://trello.com/c/zveGP8hN
- PHP: https://trello.com/c/34kzPmkB

Also tidied up some inconsistencies with other images. Will do python & ruby in another PR that I already have work for those images in.

Ping @jhadvig for tech review on Perl

Ping @bparees for tech review on PHP

### CODE REVIEW

File: using_images/sti_images/php.adoc [hunk 1]
```diff
@@ -26,58 +33,97 @@ This image comes in two flavors, depending on your needs:
...
-****
-`$ docker pull openshift/php-55-centos7`
-****
+----
+$ docker pull openshift/php-55-centos7
+----
 
-To use these images, you can either access them directly from those registries or push them into your OpenShift docker registry.  In addition, it is recommended that you create an ImageStream that points to the image, either in your docker registry or at the external location.  Your OpenShift resources can then reference the ImageStream.  You can find example ImageStream definitions for all the provided OpenShift images https://github.com/openshift/origin/tree/master/examples/image-streams[here].
+To use these images, you can either access them directly from these
+registries, or push them into your OpenShift docker registry. Additionally,
```
Comment (bparees) [inline using_images/sti_images/php.adoc]: no comma needed after registries.

no comma needed after "to use these images" either i don't think.

Comment (tpoitras) [inline using_images/sti_images/php.adoc]: Thanks Ben

**Comment by tpoitras** [comment]
Ping @jhadvig for tech review on Perl

File: using_images/sti_images/perl.adoc [hunk 2]
```diff
@@ -10,10 +10,18 @@
 toc::[]
 
 == Overview
-OpenShift provides https://github.com/openshift/source-to-image[STI] enabled Perl images for building and running Perl applications.  These images can be used with the link:../../architecture/core_objects/builds.html#sti-build[STI build strategy].
+OpenShift provides https://github.com/openshift/source-to-image[STI] enabled
+Perl images for building and running Perl applications. These images can be used
+with the
+link:../../architecture/core_objects/builds.html#sti-build[STI build strategy];
```
Comment (jhadvig) [inline using_images/sti_images/perl.adoc]: shouldnt this be ended with dot ? It looks kinda weird with semicolon

**Comment by jhadvig** [comment]
@tpoitras just a small nit,otherwise LGTM for the Perl part

File: using_images/sti_images/perl.adoc (hunk 2 above)
Comment (tpoitras) [inline using_images/sti_images/perl.adoc]: @jhadvig -- No. :) The semicolon indicates that what follows is a continuation of the sentence. A semicolon is used between two closely related independent clauses.

**Comment by tpoitras** [comment]
Thank you, @jhadvig

**Comment by tpoitras** [comment]
ping for Tech Review! @bfallonf @adellape @tnguyen-rh

File: using_images/db_images/mysql.adoc [hunk 3]
```diff
@@ -26,42 +29,53 @@ This image comes in two flavors, depending on your needs:
...
 == Configuration and Usage
 
 === Initializing the Database
 
-The first time you use the shared volume, the database is created along with the database administrator user and the MySQL root user (if you specify the `*MYSQL_ROOT_PASSWORD*` environment variable).  Afterwards, the MySQL daemon starts up. If you are re-attaching the volume to another container then the database, database user and the administrator user are not created, and the MySQL daemon starts.
+The first time you use the shared volume, the database is created along with 
+the database administrator user and the MySQL root user (if you specify the 
+`*MYSQL_ROOT_PASSWORD*` environment variable).  Afterwards, the MySQL daemon 
+starts up. If you are re-attaching the volume to another container then the 
+database, database user and the administrator user are not created, and the
```
Comment (bfallonf) [inline using_images/db_images/mysql.adoc]: s/database, database user and the administrator user/database, database user, and the administrator user

File: using_images/sti_images/perl.adoc (hunk 2 above)
Comment (tpoitras) [inline using_images/sti_images/perl.adoc]: It appears I've been using the semicolon incorrectly sometimes. Thanks @jhadvig and @bfallonf for highlighting this.

**Comment by adellape** [comment]
@tpoitras I merged https://github.com/openshift/openshift-docs/pull/408 since the related code changes had already landed. It was a small change to the MongoDB topic, but you'll need to rebase this.

**Comment by tpoitras** [comment]
@adellape -- rebased! :)

File: using_images/sti_images/perl.adoc [hunk 4]
```diff
@@ -10,10 +10,18 @@
...
+the https://github.com/openshift/sti-perl[STI builder image] assembles your
+application source with any required dependencies to create a new image
+containing your Perl application. This resulting image can be run either by
+OpenShift or by Docker.
 
 == Versions
-Currently, OpenShift only supports version https://github.com/openshift/sti-perl/tree/master/5.16[5.16] of Perl.
+Currently, OpenShift supports version
+Ghttps://github.com/openshift/sti-perl/tree/master/5.16[5.16] of Perl.
```
Comment (adellape) [inline using_images/sti_images/perl.adoc]: Wayward "G" here breaks the link.

**Comment by adellape** [comment]
@tpoitras Other than the one comment in the Perl topic, :+1:.

I also made a couple minor suggestions about some general cross-linking in the Python/Ruby PR (https://github.com/openshift/openshift-docs/pull/353#issuecomment-107470457), in case you like any of those and want to apply them similarly in these topics.

**Comment by tpoitras** [comment]
@adellape -- good spotting on that errand letter. Thank you. I've also applied those links here. Should be good now, I hope!

**Comment by adellape** [comment]
Great, thanks. :poultry_leg: Merging.

**Pull request merged**